src/builder/modules/Generator.res

```diff
@@ -122,8 +122,49 @@ let usePage = id => {
   model := Builder.usePage(model.contents, id)
 }
 
-let fillBackgroundColor = (fillStyle: string) => {
-  model := Generator_Builder.fillBackgroundColor(model.contents, fillStyle)
+let fillBackgroundColor = (color: string) => {
+  model := Generator_Builder.fillBackgroundColor(model.contents, color)
+}
+
+let fillBackgroundColorWithWhite = () => {
+  model := Generator_Builder.fillBackgroundColor(model.contents, "#ffffff")
+}
+
+let drawLine = (
+  from: Builder.position,
+  to: Builder.position,
+  ~color: string="#000000",
+  ~width: float=1.0,
+  ~pattern: array<int>=[],
+  ~offset: int=0,
+  (),
+) => {
+  model := Generator_Builder.drawLine(model.contents, from, to, ~color, ~width, ~pattern, ~offset)
+}
+
+let drawFold = (from: Builder.position, to: Builder.position) => {
+  model :=
+    Generator_Builder.drawLine(
+      model.contents,
+      from,
+      to,
+      ~color="#7b7b7b",
+      ~width=1.0,
+      ~pattern=[2, 2],
+      ~offset=3,
```

Comment:
I'm good with this. Something that I was wondering is if the offset should be a function of the line length? No need to change, just something I was thinking about.

Reply:
Eventually, the offset will need to be in certain cases, but for now all lines drawn just need an offset of 3.

Reply:
Got it, I see how this works now:

```
Position: 0123
Pattern:  XXOO
```

So an offset of 3 starts the fold line with that last single blank pixel, and then restarts.

This should work well with the common line lengths most people are using today.